docs/examples/bifacial/plot_bifi_model_pvwatts.py

```diff
@@ -0,0 +1,111 @@
+"""
+Bifacial Modeling - procedural
+==============================
+
+Example of bifacial modeling using procedural method
+"""
+
+# %%
+# This example shows how to complete a bifacial modeling example using the
+# :py:func:`pvlib.pvsystem.pvwatts_dc` with the
+# :py:func:`pvlib.bifacial.pvfactors_timeseries` function to transpose
+# GHI data to both front and rear Plane of Array (POA) irradiance.
+
+import pandas as pd
+from pvlib import location
+from pvlib import tracking
+from pvlib import bifacial
+from pvlib import temperature
+from pvlib import pvsystem
+import matplotlib.pyplot as plt
+
+# using Greensboro, NC for this example
+lat, lon = 36.084, -79.817
+tz = 'Etc/GMT+5'
+times = pd.date_range('2021-06-21', '2021-06-22', freq='1T', tz=tz)
+
+# create location object and get clearsky data
+site_location = location.Location(lat, lon, tz=tz, name='Greensboro, NC')
+cs = site_location.get_clearsky(times)
+
+# get solar position data
+solar_position = site_location.get_solarposition(times)
+
+# set ground coverage ratio and max_angle to
+# pull orientation data for a single-axis tracker
+gcr = 0.35
+max_phi = 60
+orientation = tracking.singleaxis(solar_position['apparent_zenith'],
+                                  solar_position['azimuth'],
+                                  max_angle=max_phi,
+                                  backtrack=True,
+                                  gcr=gcr
+                                  )
+
+# set axis_azimuth, albedo, pvrow width and height, and use
+# the pvfactors  engine for both front and rear-side absorbed irradiance
+axis_azimuth = 180
+pvrow_height = 3
+pvrow_width = 4
+albedo = 0.2
+irrad = bifacial.pvfactors_timeseries(solar_position['azimuth'],
+                                      solar_position['apparent_zenith'],
+                                      orientation['surface_azimuth'],
+                                      orientation['surface_tilt'],
+                                      axis_azimuth,
+                                      cs.index,
+                                      cs['dni'],
+                                      cs['dhi'],
+                                      gcr,
+                                      pvrow_height,
+                                      pvrow_width,
+                                      albedo
+                                      )
+
+# turn into pandas DataFrame
+irrad = pd.concat(irrad, axis=1)
+
+# using bifaciality factor and pvfactors results, create effective
+# irradiance data
+bifaciality = 0.75
+effective_irrad_bifi = irrad['total_abs_front'] + (irrad['total_abs_back']
+                                                   * bifaciality)
+
+
+# get cell temperature using the Faiman model
+# it is worth noting that this example uses total frontside incident
+# irradiance, however, for bifacial modeling, total irradiance (inclusive of
+# front and rear side) may be needed.
```

Comment:
The wording of this comment undercuts the example a bit. To avoid the question, it may be better to input `effective_irrad_bifi` rather than `irrad['total_inc_front']`. AFAIK, there is no cell temperature model specific to bifacial modules. [This paper](https://ieeexplore.ieee.org/document/8366045) found that bifacial and monofacial module temperatures are predicted equally well by one of the available cell temperature models, so it's not a stretch to assume that Faiman applies also to bifacial models.